packages/legend-application-studio/src/stores/sidebar-state/LocalChangesState.ts

```diff
@@ -822,9 +822,11 @@ export class TextLocalChangesState extends LocalChangesState {
   }
 
   *restartChangeDetection(): GeneratorFn<void> {
-    this.editorStore.changeDetectionState.computeLocalChangesInTextMode(
-      this.editorStore.changeDetectionState.workspaceLocalLatestRevisionState
-        .entities,
-    );
+    yield Promise.all([
```

Comment:
do you need this `yield Promise.all()` if this is the only function you call? Also, looks like this function does not need to be a generator, can we change to have it as an `async` function?